croncert-config.yml

```diff
@@ -356,6 +356,145 @@ scrapers:
+    fields:
+      - name: location
+        value: The Empty Bottle
+      - name: city
+        value: Chicago
+      - name: "type"
+        value: "concert"
+      - name: "sourceUrl"
+        value: "https://www.emptybottle.com"
+      - name: "url"
```

Comment:
added `url` field because it's mandatory for the event-api and croncert. As you know this would normally point to the event's specific website but in this case unfortunately it's not possible. Still there needs to be some value.

Reply:
Oops.I missed that one. Sorry.

Reply:
no worries :)